### DISCUSSION TITLE
مصطلح hydration

### DISCUSSION THREAD

**Discussion by boussadjra** [vuejs-translations/docs-ar#79] [مصطلحات] [answered]
لم أجد في النت مصطلحا تقنيا لكلمة hydration و التي تعني بشكل عام الإماهة و الترطيب، وتقنيا تعني إضافة شيفرة برمجية لجعل الـHTML الساكن تفاعليا.
حاليا اعتمدت كلمة "إماهة" لأني لا أرى حرجا في استعمالها رغم أنها تبدو مبهمة في انتظار الاتفاق على مصطلح جديد

**Comment by Abdelaziz18003** [comment]
هذا مصطلح فكرت في السابق ولم أجد له ترجمة تقنية، بعد قراءة معنى ال hydration وفهمه، اهتديت إلى مصطلح أظنه يعبر عن المعنى ألا وهو "الإنعاش"،
بعد تحويل شفرة Vue على الخادم إلى شفرة HTML وإرسالها للمتصفح، تكون الصفحة ميتة وغير قابلة للاستخدام والتفاعل معها، ال hydration تعني تقنيا إعادة وصل تطبيق Vue بالصفحة المتلقاة من الخادم لإحياءها وجعلها قابلة للاستخدام والتفاعل معها وهو مثل إضافة الماء لشجرة أو نبات فيحيى بعد أن كان ميتا، لهذا استخدموا كلمة ال hydration للتعبير عن هذا المعنى، بحثت في العربية فوجدت كلمة إنعاش كلمة مفهومة ومنتشرة وتعبر عن المعنى المطلوب.

**Reply by boussadjra** [reply]
شكرا، هذه الكلمة هي الأصوب.

**Comment by Abdelaziz18003** [comment, marked as answer]
الكلمة التي اتُّفِق عليها هي "إنعاش"
